Renderer.cs

```diff
@@ -424,3 +426,32 @@ public static void DrawLine(Vector2 start, Vector2 end, Color color, int thickne
 
 #endregion
 }
+public static class ArabicFixerSafe
+{
+    private static readonly Regex PlaceholderRegex = new(@"{\d+}", RegexOptions.Compiled);
+
+    public static string Fix(string input)
```

Comment:
Can you rename the Fix method with a better name? something like `FixXXXX`

There are 3 kinds of fixes right now